src/enigma2/Admin.h

```diff
@@ -25,16 +25,23 @@
 #include <vector>
 
 #include "utilities/DeviceInfo.h"
+#include "utilities/DeviceSettings.h"
 
 #include "libXBMC_pvr.h"
 
 namespace enigma2
 {
+
+  static const std::string ADDON_SETTINGS_XML_FILEPATH = "special://userdata/addon_data/pvr.vuplus/settings.xml";
```

Comment:
Yikes! I don't think that internal path structure and file names is part of our public "API".